### DISCUSSION TITLE
Did we just add over 40 PiB to the Network over night?

### DISCUSSION THREAD

**Discussion by Teledyol** [Chia-Network/chia-blockchain#1736] [General]
Yesterday, we were under 200, now were 241.79…

That's wot Ima seeing.

**Comment by dorkmo** [comment]
it can jump around a bit, but smoothed out it seems pretty steady https://www.chiaexplorer.com/charts/netspace i calculate about 7.5PB a day. and it looks like https://hpool.com/statistics/chia is adding about 3 of those PB a day. not sure if its possible to transfer existing plots to their pool, so it may be less "new" plots accounted from them.

**Reply by Teledyol** [reply]
Ya around 2:30 a PST it was the largest increase that I've seen. It's back to that number now 3p

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
